agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -576,10 +576,21 @@ async def list_bases_for_app_id(
     app_id: str, base_name: Optional[str] = None, **kwargs: dict
 ) -> List[VariantBaseDB]:
     assert app_id is not None, "app_id cannot be None"
-    query_expressions = VariantBaseDB.app.id == ObjectId(app_id)
     if base_name:
-        query_expressions += query_expressions(VariantBaseDB.base_name == base_name)
-    bases_db = await VariantBaseDB.find(query_expressions).sort("base_name").to_list()
```

Comment:
I don't see a difference in the logic here. Is it that the query_expressoins addition don't work?

Reply:
I found a better way to do this.